docs/templates/tableau-server-flow-template.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Tableau Server (Flow web authoring) Template
+---
+
+1. Download the Connector file (.taco).
+2. Move the .taco file to: `[Tableau_Server_Installation_Directory]/data/tabsvc/flowqueryservice/Connectors`
```

Comment:
For a multi-node Tableau server, the taco needs to be placed in the correct folders for each server node, otherwise the connector won't be installed properly.